src/casync-http.c

```diff
@@ -499,6 +529,10 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_verbose = true;
                         break;
 
+                case 'l':
+                        arg_rate_limit = strtol(optarg, NULL, 10);
```

Comment:
please do not use strtol()... it's very hard to use properly. Please use parse_size() here too and generate an error message in invalid input...

Also, I think this should accept a special value "off" or so, for turning this off...